heat/examples/vm_two_nics_with_share.yaml

```diff
@@ -0,0 +1,89 @@
+heat_template_version: 2018-03-02
+
+description: > 
+    This template illustrates the creation of a VM
+    together with a NFS share
+
+parameters:
+    user_key:
+        type: string
+        label: ssh_user_key
+        description: Public user ssh key to be injected in the cluster VMs
+        constraints:
+            - custom_constraint: nova.keypair
+    user_net1:
+        type: string
+        label: user_net1
+        description: Add the required VM network
+        constraints:
+            - custom_constraint: neutron.network
+    user_net2:
+        type: string
+        label: user_net2
+        description: Add the required VM network
+        constraints:
+            - custom_constraint: neutron.network
+    user_share_name:
+        type: string
+        label: share_name
+        description: Give a name to the share
+        constraints:
+            - length: { min: 1, max: 32 }
+              description: Share name must contain between 1 and 32 characters
+            - allowed_pattern: "[a-zA-Z0-9_]+"
+              description: Only alpha-numeric and "_" characters allowed
+    user_share_size:
+        type: number
+        label: share_size
+        description: Specify the share size (in GB)
+        constraints:
+            - range: { min: 1, max: 100 }
+    user_vm_flavor:
+        type: string
+        label: vm_flavor
+        description: Choose a flavor
+        constraints:
+            - custom_constraint: nova.flavor
+        default: m1.small
+    user_vm_image:
+        type: string
+        label: vm_image
+        description: Choose an image
+        constraints:
+            - custom_constraint: glance.image
+
+resources:
+    my_server:
+        type: OS::Nova::Server
+        depends_on:
+            - my_share
+        properties:
+          name: My_VM_with_NFS_share
+          metadata:
+              # Some metadata examples to be included in the VM
+              server: master
+              color: red
+              share_export_loc_path: { get_attr: [my_share, export_locations, 0, path] } 
```

Comment:
Replace this line by:
`
share_export_loc_path: { get_attr: [my_share, export_locations] } `

I have tested this and it works... it gives you the NFS mount path (it is not clear why the path is not there)

Reply:
Indeed, it works the way you've indicated.

Reply:
@StephaneGerardVUB could you please add this into `user_data`   section to mount the share automatically?

Reply:
instead of ubuntu user (could be any image) we should change that by root user